libs/feature/editor/src/lib/components/record-form/record-form.component.ts

```diff
@@ -8,19 +8,38 @@ import {
   EditorFieldWithValue,
   EditorSectionWithValues,
 } from '../../+state/editor.models'
+import { getGlobalConfig } from '@geonetwork-ui/util/app-config'
+import { UiInputsModule } from '@geonetwork-ui/ui/inputs'
 
 @Component({
   selector: 'gn-ui-record-form',
   templateUrl: './record-form.component.html',
   styleUrls: ['./record-form.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
-  imports: [CommonModule, FormFieldComponent, TranslateModule],
+  imports: [CommonModule, FormFieldComponent, TranslateModule, UiInputsModule],
 })
 export class RecordFormComponent {
+  isHidden: boolean
+
+  get config() {
+    console.log(getGlobalConfig().LICENSES)
+    return getGlobalConfig().LICENSES
+  }
   constructor(public facade: EditorFacade) {}
 
-  handleFieldValueChange(model: string, newValue: EditorFieldValue) {
+  onOpenDataToggled(args) {
```

Comment:
Yes, true!

Reply:
Actually, not sure I agree entirely. I think the logic of setting `isHidden` has to kept here right? because it's where we decide if the field is going to be displayed or not, so the information has to come here in the form of an Output function. WDYT?